### PR TITLE
Use CollectionUtils.isEmpty make code clean

### PR DESCRIPTION
**Pull request by zhenxianyimeng**
user CollectionUtils.isEmpty to replace code judge an array is empty or not
make code more clean and easy to understand

### CODE REVIEW

**Comment by rjernst** [comment]
@zhenxianyimeng Thank you for the PR. We have a common practice of using `== false` instead of `!` as the former is much easier to visually identify when scanning through code. Could you please update the boolean conditions in this PR with that in mind?

**Comment by zhenxianyimeng** [comment]
> @zhenxianyimeng Thank you for the PR. We have a common practice of using `== false` instead of `!` as the former is much easier to visually identify when scanning through code. Could you please update the boolean conditions in this PR with that in mind?

ok i will update it

**Comment by zhenxianyimeng** [comment]
> @zhenxianyimeng Thank you for the PR. We have a common practice of using `== false` instead of `!` as the former is much easier to visually identify when scanning through code. Could you please update the boolean conditions in this PR with that in mind?

@rjernst  i have solved it, please help me review

File: client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java [hunk 1]
```diff
@@ -715,7 +716,7 @@ public void onFailure(Exception e) {
             String scrollId = searchResponse.getScrollId();
             SearchHit[] searchHits = searchResponse.getHits().getHits();
 
-            while (searchHits != null && searchHits.length > 0) { // <2>
+            while (CollectionUtils.isEmpty(searchHits) == false) { // <2>
```
Comment (pugnascotia) [inline client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java:719]: @rjernst is it an issue that this line of code is referenced in the documentation, due to the `tag::search-scroll-example`?

Comment (rjernst) [reply client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java:719]: Yes and no. The examples are just that: examples. We don't show the full file, so they already lack anything we include.  However, while CollectionUtils is currently brought in by the HLRC through its server jar dependency, I agree we shouldn't be leaking this method in our docs.

@zhenxianyimeng Can you please revert this and any other *DocumentationIT tests?

**Comment by rjernst** [comment]
@elasticmachine ok to test

File: client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java (hunk 1 above)
Comment (zhenxianyimeng) [reply client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java:719]: @rjernst ok

Comment (zhenxianyimeng) [reply client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java]: @rjernst ok

**Review by rjernst — APPROVED** [review]
LGTM

**Comment by rjernst** [comment]
Thanks for the contribution @zhenxianyimeng.

**Pull request merged**